### ISSUE TITLE
Are you interested in collaborating? 

### ISSUE THREAD

**Issue by redthing1** [Kiffolisk/godotheads#2]
I also want to help restore the blockheads. There are other people who also seem interested. If you want I can get in touch with everyone and we can form a group to multiply all our efforts

**Comment by Kiffolisk** [comment]
yeah sure, i've already reversed a bit of code and made some progress but i forgot to commit it here
